concent_api/concent_api/settings/base.py

```diff
@@ -220,6 +220,8 @@
 # A global constant defining the length of the time window within which a requestor can verify subtask.
 SUBTASK_VERIFICATION_TIME = 4 * 3600  # 4 hours
 
+ADDITIONAL_VERIFICATION_CALL_TIME = 4 * 3600  # 4 hours
```

Comment:
Please add comment above, as there is in every other Concet custom setting.

Reply:
done

Reply:
I don't see it :) Probably you still have it locally. That's why I am doing two things:
1. :+1: - for marking it for myself as done, or 'accepted/agreed to do'. When you scroll through comments you can easily see visually which one you have already resolved.
2. Add 'done' when it is pushed to github.